themes/bootstrap3/templates/search/bulk-action-buttons.phtml

```diff
@@ -1,25 +1,25 @@
 <? if((isset($this->showBulkOptions)  && $this->showBulkOptions)
    || (isset($this->showCartControls) && $this->showCartControls)): ?>
-  <div class="bulkActionButtons hidden-print">
+  <div class="bulkActionButtons hidden-print<?if($this->idPrefix):?> form-inline<? endif; ?>">
```

Comment:
Looks like you've changed most of the new idPrefix checks to formAttr checks, except for the one right above these comments. Is that intentional or a typo?